relayer/query.go

```diff
@@ -409,7 +409,7 @@ func (c *Chain) QueryUnbondingPeriod() (time.Duration, error) {
 func (c *Chain) QueryUpgradedClient(height int64) (*codectypes.Any, []byte, clienttypes.Height, error) {
 	req := upgradetypes.QueryCurrentPlanRequest{}
 
-	queryClient := upgradetypes.NewQueryClient(c.CLIContext(height))
+	queryClient := upgradetypes.NewQueryClient(c.CLIContext(0))
```

Comment:
probably should switch these to specified height later, but doesn't need to block this PR.

Reply:
I'd prefer to leave unless we manually test the correct parameter to use